pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -509,3 +492,27 @@ func getCachedRepoInfo(upgrade *harvesterv1.Upgrade) (*RepoInfo, error) {
 	}
 	return repoInfo, nil
 }
+
+func isVersionUpgradable(currentVersion, minUpgradableVersion string) error {
+	if minUpgradableVersion == "" {
+		logrus.Debug("No minimum upgradable version specified, continue the upgrading")
+	} else {
```

Comment:
```suggestion
	if minUpgradableVersion == "" {
		logrus.Debug("No minimum upgradable version specified, continue the upgrading")
		return nil
	}
```

Suggest returning here and move the else block out.

Reply:
Updated, thanks!